src/to_xml.jl

```diff
@@ -96,8 +96,10 @@ function to_xml(ts::AbstractTestSet)
     total_ntests = 0
     total_nfails = 0
     total_nerrors = 0
-    x_testcases = map(ts.results) do result
+    x_testcases = map(enumerate(ts.results)) do (i, result)
```

Comment:
See comment below

```suggestion
    x_testcases = map(ts.results) do result
````